apps/src/studio/studio.js

```diff
@@ -1170,6 +1170,14 @@ Studio.willSpriteTouchWall = function (sprite, xPos, yPos) {
   return Studio.willCollidableTouchWall(sprite, xCenter, yCenter);
 };
 
+Studio.getWallValue = function (row, col) {
+  if (Studio.walls) {
+    return skin[Studio.walls] ? (skin[Studio.walls][row][col] << constants.WallCoordsShift): 0;
```

Comment:
we shift them into place here so that either SquareType.WALL style walls or row/col specific walls can be returned. But this allows the skin to define them without shifting.